app/jobs/create_push_notifications_job.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePushNotificationsJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(event_id:)
+    event = ::Event.find_by_id(event_id)
+
+    raise 'A valid Event must be provided' unless event
+
+    ::PushNotifications::Creator.create!(event: event)
```

Comment:
Do we really need the leading `::`? This reference lives in the global namespace so I don't see why `PushNotifications::Creator` would have any problem finding the constant.

Reply:
I'm just used to it, it makes it easier to ruby to find constants. Especially in Rails the constants lookup can be really "kafkian"...